### PR TITLE
WIP: Update LKJ notebook to not refer to Wishart

### PR DESCRIPTION
**Pull request by zaxtax**
As the Wishart is highly discouraged now, including the Wishart-Bartlett form, we should change the documentation to not refer to it, and even perhaps talk more about the LKJ distribution.

### CODE REVIEW

**Comment by junpenglao** [comment]
> As the Wishart is highly discouraged now, including the Wishart-Bartlett form

Do you have a reference for that? I thought Bartlett form is ok.

**Comment by zaxtax** [comment]
The warning on line https://github.com/pymc-devs/pymc3/blob/master/pymc3/distributions/multivariate.py#L550 seems like a pretty strong nudge against the Bartlett form.

Maybe instead of saying the Bartlett form is highly discouraged, the LKJ is highly encouraged over the Wishart-Bartlett.

File: docs/source/notebooks/LKJ.ipynb [hunk 1]
```diff
@@ -13,9 +13,7 @@
...
-    "\n",
-    "The normal-Wishart prior is conjugate for the multivariate normal model, so we can find the posterior distribution in closed form.  Even with this closed form solution, sampling from a multivariate normal model in `pymc3` is important as a building block for more complex models.\n",
+    "Outside of the [beta](https://en.wikipedia.org/wiki/Beta_distribution)-[binomial](https://en.wikipedia.org/wiki/Binomial_distribution) model, the multivariate normal model is likely the most studied Bayesian model in history. `Pymc3` supports sampling from the [LKJ distribution](http://www.sciencedirect.com/science/article/pii/S0047259X09000876). The LKJ distribution represents the distribution on correlation matrices and is conjugate to the multivariate normal distribution. This post will show how to fit a simple multivariate normal model using `pymc3` with a normal-LKJ prior.\n",
```
Comment (twiecki) [inline docs/source/notebooks/LKJ.ipynb]: `Pymc3` -> `PyMC3`

**Comment by twiecki** [comment]
Thanks @zaxtax. I agree that we should change the doc strings too.

@junpenglao Wishart is pretty terrible (which form does not matter here): https://dahtah.wordpress.com/2012/03/07/why-an-inverse-wishart-prior-may-not-be-such-a-good-idea/

**Comment by zaxtax** [comment]
Just to clarify, should I also update the doc strings on Wishart-Bartlett to nudge people to LKJ there as well?

**Comment by junpenglao** [comment]
> Just to clarify, should I also update the doc strings on Wishart-Bartlett to nudge people to LKJ there as well?

Yes you should do that as well then. 
Actually, in that case maybe we should advice user to use `LKJCholeskyCov`, since currently it is the fastest one? @twiecki

**Pull request merged**